### PR TITLE
30881 Feat: Move gordo client code

### PR DESCRIPTION
**Pull request by rkicz**

### CODE REVIEW

File: gordo/__init__.py [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import logging
-import os
 import traceback
 from typing import Tuple
 import warnings
+from pkgutil import extend_path
+
+
+__path__ = extend_path(__path__, __name__)  # type: ignore
```
Comment (koropets) [inline gordo/__init__.py:10]: Please add a comment on why do we need this here

File: requirements/requirements.in [hunk 2]
```diff
@@ -30,3 +30,4 @@ gordo-dataset~=2.3.0
 jeepney>=0.6
 packaging~=20.7
 pydantic~=1.7.3
+gordo.client~=0.1.5
```
Comment (koropets) [inline requirements/requirements.in]: Lets it be a future version:
```suggestion
gordo.client~=0.2.0
```

File: tests/gordo/client/test_forwarders.py [hunk 3]
```diff
@@ -102,9 +103,10 @@ def test_influx_forwarder(influxdb, influxdb_uri, sensors, sensors_str):
...
     df = get_test_data(pd.MultiIndex.from_tuples(input_keys + output_keys))
 
+    client_machine = ClientMachine(**machine.to_dict())
     # Create the forwarder and forward the 'predictions' to influx.
     forwarder = ForwardPredictionsIntoInflux(destination_influx_uri=influxdb_uri)
-    forwarder.forward_predictions(predictions=df, machine=machine)
+    forwarder.forward_predictions(predictions=df, machine=client_machine)
 
     # Client to manually verify the points written
     client = influx_client_from_uri(influxdb_uri, dataframe_client=True)
```
Comment (koropets) [inline tests/gordo/client/test_forwarders.py]: This whole  test could also be in `gordo-client` package

File: tests/gordo/client/test_client.py [hunk 4]
```diff
@@ -18,6 +18,7 @@
 from mock import patch, call
```
Comment (koropets) [inline tests/gordo/client/test_client.py:19]: Please add TODO comment here for moving this test into `gordo-client` package in the future

File: gordo/__init__.py (hunk 1 above)
Comment (rkicz) [reply gordo/__init__.py:10]: fixed

File: requirements/requirements.in (hunk 2 above)
Comment (rkicz) [reply requirements/requirements.in]: fixed

File: tests/gordo/client/test_forwarders.py (hunk 3 above)
Comment (rkicz) [reply tests/gordo/client/test_forwarders.py]: I'l take a look on separate PR

**Pull request merged**